var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -18,6 +18,9 @@ class Julia(Package):
     maintainers = ['glennpj']
 
     version('master', branch='master')
+    version('1.5.1', tag='v1.5.1')
+    version('1.5.0', tag='v1.5.0')
+    version('1.4.2', tag='v1.4.2')
     version('1.4.1', sha256='b21585db55673ac0668c163678fcf2aad11eb7c64bb2aa03a43046115fab1553')
```

Comment:
Can you use the tarballs so that we have the sha256 as part of the version?

Reply:
Done.